.goreleaser.yaml

```diff
@@ -0,0 +1,113 @@
+project_name: rill
+
+release:
+  prerelease: auto
+  draft: true
+
+builds:
+  - id: darwin_amd4
+    main: ./cli/main.go
+    binary: rill
+    goos:
+      - darwin
+    goarch:
+      - amd64
+    env:
+      - PKG_CONFIG_SYSROOT_DIR=/sysroot/macos/amd64
+      - PKG_CONFIG_PATH=/sysroot/macos/amd64/usr/local/lib/pkgconfig
+      - CC=o64-clang
+      - CXX=o64-clang++
+    flags:
+      - "-mod=readonly"
+    ldflags:
+      - -s -w -X main.Version={{.Version}} -X main.Commit={{.ShortCommit}} -X main.BuildDate={{.Date}}
+
+  - id: darwin_arm64
+    main: ./cli/main.go
+    binary: rill
+    goos:
+      - darwin
+    goarch:
+      - arm64
+    env:
+      - PKG_CONFIG_SYSROOT_DIR=/sysroot/macos/amd64
+      - PKG_CONFIG_PATH=/sysroot/macos/amd64/usr/local/lib/pkgconfig
+      - CC=oa64-clang
+      - CXX=oa64-clang++
+    flags:
+      - "-mod=readonly"
+    ldflags:
+      - -s -w -X main.Version={{.Version}} -X main.Commit={{.ShortCommit}} -X main.BuildDate={{.Date}}
+
+  - id: linux
+    main: ./cli/main.go
+    binary: rill
+    goos:
+      - linux
+    goarch:
+      - amd64
+    env:
+      - CC=x86_64-linux-gnu-gcc
+      - CXX=x86_64-linux-gnu-g++
+      - CGO_CFLAGS=--sysroot=/sysroot/linux/amd64
+      - CGO_LDFLAGS=--sysroot=/sysroot/linux/amd64
+      - PKG_CONFIG_SYSROOT_DIR=/sysroot/linux/amd64
+      - PKG_CONFIG_PATH=/sysroot/linux/amd64/usr/local/lib/pkgconfig
+    flags:
+      - "-mod=readonly"
+    ldflags:
+      - -s -w -X main.Version={{.Version}} -X main.Commit={{.ShortCommit}} -X main.BuildDate={{.Date}}
+
+blobs:
+  - provider: gs
+    bucket: stage-cdn.rilldata.io #TODO: testing
+    folder: "rill/v{{ .Version }}"
+    extra_files:
+      - glob: ./dist/**/*
+
+  - provider: gs
+    bucket: stage-cdn.rilldata.io #TODO: testing
+    folder: "rill/latest"
+    extra_files:
+      - glob: ./dist/**/*
+
+brews:
+  - homepage: https://github.com/rilldata/rill-developer
+    description: The Rill CLI
+    name: rill
+    license: Apache 2.0
+    url_template: https://cdn.rilldata.io/rill/{{ .Tag }}/{{ .ArtifactName }} #TODO: testing
+    tap:
+      owner: rilldata
+      name: homebrew-rill-developer
+      branch: main
+      token: "{{ .Env.GITHUB_TOKEN }}"
+    test: |
+      system "#{bin}/rill --version"
+    install: |
+      bin.install "rill"
+    commit_author:
+      name: rilldata
+      email: support@rilldata.com
+
+archives:
+  - replacements:
+      darwin: Darwin
+      linux: Linux
+    files:
+      - README.md
+      - LICENSE*
+    name_template: "{{ .ProjectName }}_{{ .Os }}_{{ .Arch }}"
```

Comment:
This is producing binary names with uppercase OS, see the screenshot. They should be all lowercase, e.g. `rill_darwin_arm64.tar.gz`

Also: Is there a way to output `.zip` files instead of `.tar.gz`? It's just more common/easy for people

<img width="1022" alt="Screenshot 2022-12-05 at 11 35 23" src="https://user-images.githubusercontent.com/791356/205616446-f4cea0f5-3a15-423c-94c5-fd335e7620ba.png">

Reply:
@begelundmuller sure making it lowercase, 
But on you second comment, what i found is mostly for mac and linux everyone prefers .tar.gz Only for window good to have .zip

Reply:
@rakeshsharma14317 can you find out how to do `zip` and then will discuss options with product. But I think they'll prefer `.zip`, it's now very common on unix systems. E.g. duckdb only publishes `.zip` archives